src/Http/Controllers/AppointmentController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace LibreEHR\FHIR\Http\Controllers;
+
+use Illuminate\Http\Request;
+use LibreEHR\FHIR\Adapters\FHIRAppointmentAdapter;
+use Illuminate\Support\Facades\DB;
+use phpDocumentor\Reflection\DocBlock\Tags\Return_;
+
+class AppointmentController extends AbstractController
+{
+    public function __construct()
+    {
+        $this->adapter = new FHIRAppointmentAdapter;
+    }
+
+    public function index()
+    {
+
+    }
+
+    public function getSchedule()
+    {
+        $results = DB::select('select * from openemr_postcalendar_events');
```

Comment:
Schedule should be its own resource endpoint with its own controller, and adapter.